modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/SourceProcessorTestParameters.java

```diff
@@ -40,6 +40,14 @@ public SourceProcessorTestParameters addExpectedMessage(
 		return this;
 	}
 
+	public SourceProcessorTestParameters addExpectedMessage(String[] messages) {
```

Comment:
Hey @nicolas-sss, I would rename this to `addExectedMessages()` since we are dealing with multiple messages.